librocksdb_sys/crocksdb/c.cc

```diff
@@ -2936,6 +2936,17 @@ const char* crocksdb_pinnableslice_value(const crocksdb_pinnableslice_t* v,
   return v->rep.data();
 }
 
+size_t crocksdb_get_supported_compression_number() {
+  return rocksdb::GetSupportedCompressions().size();
+}
+
+void crocksdb_get_supported_compression(unsigned char* v) {
```

Comment:
Should add a `len` parameter to indicate the expected length of the result, and validate in this function.